cmd/influxd/config.go

```diff
@@ -40,6 +40,10 @@ type Config struct {
 	Version           string `toml:"-"`
 	InfluxDBVersion   string `toml:"-"`
 
+	Authentication struct {
```

Comment:
I think we'll need a section for Authentication. It seems like an area that will grow and I'd rather not prefix a bunch of fields with "auth-".